src/Input/SubLabel.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { variant as variantBuilder } from 'styled-system'
+
+import { Typography } from '../Typography'
+import { default as variants } from './SubLabel.variants'
+
+export const SubLabel = ({ variant, content, preserveSpace }) => (
+  <Wrapper preserveSpace={preserveSpace} variant={variant}>
+    {content}
+  </Wrapper>
+)
+
+const Wrapper = styled(Typography)`
+  font-size: 11px;
+  line-height: 16px;
```

Comment:
Use these values in variants. This way, the values can be easily changes by adding a new variant.

Reply:
Done